app/schemas/models/mandate.schema.coffee

```diff
@@ -39,9 +39,13 @@ module.exports = MandateSchema = {
             min: {type: 'number', minimum: 1, exclusiveMinimum: true, format: 'seconds'}
             max: {type: 'number', minimum: 5, exclusiveMinimum: true, format: 'seconds'}
     currentTournament:
-      name: 'Current Tournament'
-      description: 'The identifier of the current active tournament, if any.'
-      type: 'string'
+      c.array {description: 'The arrays of the current active tournament, if any.'},
+      c.object {},
+        level: 'string',
+        courseInstanceID: c.objectId(),
+        startAt: 'integer',
+        endAt: 'integer',
+    tournamentOnlyLevels: c.array { description: 'levels only accessible during tournament and specific course instance id'}, 'string'
```

Comment:
I don't understand this description very well. Is the specific course instance id stored here, or are these the only levels accessible by the courseInstanceID above?

Reply:
I am also confused by the indentation here. Is `currentTournament` an array of objects? And are `tournamentOnlyLevels` not tied to the `currentTournament` and instead global?

Reply:
> I don't understand this description very well. Is the specific course instance id stored here, or are these the only levels accessible by the courseInstanceID above?

Actually, one object of `currentTournament` is one specific tournament which should correspond to a specific course instance and a specific level with start time and end time.

`tournamentOnlyLevels` means the level won't be accessible beyond the course instances in `currentTournament`. In China, we don't want to open `magic-rush` to everyone, it will only open during a tournament with a specific schedule. 
This field is mainly used in 
https://github.com/codecombat/codecombat-server/blob/setup-classroom-tournament-in-mandates/server/middleware/levels.coffee#L103

While `currentTournament.level` can be any level